critiquebrainz/ws/review/views.py

```diff
@@ -267,6 +450,20 @@ def review_vote_put_handler(review_id, user):
 
     **OAuth scope:** vote
 
+    **Request Example:**
+
+    .. code-block:: bash
+
+        $ curl -X PUT -H "Content-type: application/json" -H "Authorization: Bearer <your Oauth token>" -d '{"vote":true}' "https://critiquebrainz.org/ws/1/review/9cb11424-d070-4ac1-8771-a8703ae5cccd/vote"
```

Comment:
You can also split long rows so that they are fully visible and easier to read:

```bash
$ curl "https://critiquebrainz.org/ws/1/review/9cb11424-d070-4ac1-8771-a8703ae5cccd/vote" \
       -X PUT -H "Content-Type: application/json" -d '{"vote":true}' \
       -H "Authorization: Bearer <access token>"
```